src/applications/disability-benefits/all-claims/tests/content/supportingEvidenceOrientation.unit.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import { render } from '@testing-library/react';
+import { supportingEvidenceOrientation } from '../../content/supportingEvidenceOrientation';
+
+describe('supportingEvidenceOrientation', () => {
+  it('renders increase only li if only claiming increase', () => {
```

Comment:
Converted tests look good! Great job! 

I would just update the `it()` titles. A helpful format to follow is`should X when Y`...so for example it could be something like `renders increase message when claiming an increase` or for below `renders new condition message when claiming a new condition`